### PR TITLE
Github Action: add more exempt issue labels

### LINKED ISSUE #1339: Some issues shouldn't be closed by time
The github bot has marked a lot of issues to stale and closed them automatically. However, some valuable and unresolved issues are closed. I think an issue should be marked as `stale` iff it's a bug report and we haven't received enough information to reproduce it. For other situations, like feature request, we could close it or reference to another tracking issue manually (or just left it open, reminding us to implement the feature).

Could we have some methods to mark an issue valuable to avoid it to be closed by the bot?

### PR DESCRIPTION
**Pull request by cwen0**
Signed-off-by: cwen0 <cwenyin0@gmail.com>

### What problem does this PR solve?
 

Resolve #1339.

### What is changed and how does it work? 

* Add more exempt issue labels: "type/enhancement,type/suggestion,type/bug"  
* Extend idle number of days before marking issue and pr

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by zhouqiang-cl — DISMISSED** [review]
LGTM

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Comment by g1eny0ung** [comment]
@cwen0 What about upgrade the action to `actions/stale@v3`? The `v1.1.0` is too old to find its docs.

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -15,9 +15,9 @@ jobs:
...
         stale-pr-message: 'This pr is stale because it has been open 60 days with no activity. Remove stale label or comment or this will be closed in 15 days'
-        days-before-stale: 60
-        days-before-close: 15
+        days-before-stale: 90
+        days-before-close: 21
         stale-issue-label: 'lifecycle/stale'
         stale-pr-label: 'lifecycle/stale'
-        exempt-issue-label: 'lifecycle/frozen'
-        exempt-pr-label: 'lifecycle/frozen'
+        exempt-issue-label: 'lifecycle/frozen,type/enhancement,type/suggestion,type/bug'
```
Comment (yeya24) [inline .github/workflows/stale.yml:22]: Some reported issues labeled `bug` are actually not bugs, maybe configuration or environment problems. I feel we shouldn't include it here.

Comment (cwen0) [reply .github/workflows/stale.yml:22]: If we can confirm the reported issue is not a bug, we should remove the `bug` label.

**Comment by cwen0** [comment]
> @cwen0 What about upgrade the action to `actions/stale@v3`? The `v1.1.0` is too old to find its docs.

Done

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -10,14 +10,14 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/stale@v1.1.0
+    - uses: actions/stale@v3
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         stale-issue-message: 'This issue is stale because it has been open 60 days with no activity. Remove stale label or comment or this will be closed in 15 days'
         stale-pr-message: 'This pr is stale because it has been open 60 days with no activity. Remove stale label or comment or this will be closed in 15 days'
```
Comment (g1eny0ung) [inline .github/workflows/stale.yml:17]: Still need to update messages.

- `60` => `90`
- `15` => `21`

File: .github/workflows/stale.yml [hunk 3]
```diff
@@ -10,14 +10,14 @@ jobs:
...
-        days-before-stale: 60
-        days-before-close: 15
+        days-before-stale: 90
+        days-before-close: 21
         stale-issue-label: 'lifecycle/stale'
         stale-pr-label: 'lifecycle/stale'
-        exempt-issue-label: 'lifecycle/frozen'
-        exempt-pr-label: 'lifecycle/frozen'
+        exempt-issue-label: 'lifecycle/frozen,type/enhancement,type/suggestion,type/bug'
+        exempt-pr-label: 'lifecycle/frozen,type/enhancement,type/bug-fix'
```
Comment (g1eny0ung) [inline .github/workflows/stale.yml:23]: After referring to docs, these fields should be updated:

- `exempt-issue-label` => `exempt-issue-labels`
- `exempt-pr-label` => `exempt-pr-labels`

File: .github/workflows/stale.yml (hunk 2 above)
Comment (cwen0) [reply .github/workflows/stale.yml]: Fixed

File: .github/workflows/stale.yml (hunk 3 above)
Comment (cwen0) [reply .github/workflows/stale.yml]: Fixed

File: .github/workflows/stale.yml (hunk 1 above)
Comment (yeya24) [reply .github/workflows/stale.yml]: From my experience, this is not an easy thing and it is prone to forget. If we want to enable this we need to have regular bug scrub to revisit issues. If we can ensure removing `bug` label for every wrong issue then it sounds good.

**Review by g1eny0ung — APPROVED** [review]
LGTM!

**Review by yeya24 — APPROVED** [review]
LGTM

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Pull request merged**